Source/Scene/Cesium3DTile.js

```diff
@@ -206,14 +231,17 @@ define([
         },
 
         /**
-         * Get the tight oriented bounding box
+         * Get the bounding volume of the tile
          *
-         * @type {Promise}
+         * @type {Object}
          * @readonly
          */
-        orientedBoundingBox : {
+        boundingVolume : {
```

Comment:
That's probably the best approach for now. The code I was using before for the conversion is:

`this.boundingSphere = BoundingSphere.fromOrientedBoundingBox(tile.orientedBoundingBox);`